README.md

```diff
@@ -243,43 +174,33 @@ Feel free to check the [issues page](../../issues/).
 
 <!-- SUPPORT -->
 
-## ⭐️ Show your support <a name="support"></a>
+## ⭐️ Show your support
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+If you like this project give it a star!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
+## 🙏 Acknowledgments
 
-I would like to thank...
+I would like to thank Mircroverse for the learning experience
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
+## ❓ FAQ 
 
-- **[Question_2]**
+- What is this project about?
 
-  - [Answer_2]
+  - About learning and developing skills learned at Microverse
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
-## 📝 License <a name="license"></a>
+## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] There's no file called LICENSE in your directory that you referenced here. You only have MIT.md file which you should referenced instead. At this moment, the file cannot be found. Kindly fix this.

![license](https://github.com/ommarz/Hello-Microverse/assets/26597432/62f165b4-1c22-48a0-8e69-5c8bb4f38f3c)